frontend/packages/console-shared/src/utils/resource-utils.ts

```diff
@@ -410,6 +416,21 @@ const getRolloutStatus = (
   return notFailedOrCancelled && previous && previous.pods.length > 0;
 };
 
+const deploymentIsInProgress = (resource: K8sResourceKind): boolean => {
```

Comment:
```suggestion
const isDeploymentInProgress = (resource: K8sResourceKind): boolean => {
```

Reply:
fixed in https://github.com/openshift/console/pull/3627/commits/e0d1c431754ec9c3ea4b8b29025478d761bc501f